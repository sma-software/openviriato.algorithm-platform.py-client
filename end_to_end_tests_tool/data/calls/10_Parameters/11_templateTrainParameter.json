{
    "restResource": "/parameters/{key}",
    "urlParameters": [ "templateTrainParameter" ],
    "httpMethod": "GET",
    "requestBody": null,
    "CSharpMethodName": "GetAlgorithmTrainParameter(key)",
    "CSharpMethodParameterTypes": [ "System.String" ],
    "subcategory": "Get Parameters"
}
