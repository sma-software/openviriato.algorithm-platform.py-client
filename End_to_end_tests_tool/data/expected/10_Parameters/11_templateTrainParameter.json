{
  "id": 2060,
  "trainPathNodes": [
    {
      "id": 1332,
      "sectionTrackId": null,
      "nodeId": 18,
      "nodeTrackId": null,
      "formationId": 1187,
      "arrivalTime": "2003-05-01T00:05:00",
      "departureTime": "2003-05-01T00:05:00",
      "minimumRunTime": null,
      "minimumStopTime": "P0D",
      "stopStatus": "commercialStop",
      "sequenceNumber": 0
    },
    {
      "id": 1696,
      "sectionTrackId": 1172,
      "nodeId": 10,
      "nodeTrackId": null,
      "formationId": null,
      "arrivalTime": "2003-05-01T00:10:00",
      "departureTime": "2003-05-01T00:10:00",
      "minimumRunTime": "PT5M",
      "minimumStopTime": "P0D",
      "stopStatus": "commercialStop",
      "sequenceNumber": 1
    }
  ],
  "debugString": "RVZH_1_1_J03 tt_(G)"
}